docs/installation/lti_integration.md

```diff
@@ -0,0 +1,165 @@
+}
+```
+Public key:
+```json
+{
+    "kty": "RSA",
+    "e": "AQAB",
+    "use": "sig",
+    "kid": "Nm4-PaiSvgIwk7OlFn0wK_64bt402rsHmu5zz8fTOyE",
+    "alg": "RS256",
+    "n": "jb_PoRxF0KaCMASsZwQyZFtoF6X19geXJJsx8QbAcsQEKt8RX34eAueVXn1K49UaGDkK-G9UCDmLDYTKMgjz1mtFKuV2J6CwowplkBq9rE_fUgkSY0XfLC3pCRSaQ6kjwwUjbFjF7tWVQHFhTgjqQ85HA5Pd3ix1yHnMPaNZ08CwucAx1st_WLauEmqdkmfXNIA65S5CO8EXxo94CVJ-DIZ3X7HDJq0m28SRKMR7sPM1q8A3a3z_n7DzIytjRyQkLcCWQq9oLT5dTuvAHz3Hasb1hqGqy9uS3RCFvjXk3GW3JMonVfhJ7310gUCAojEqsQ06vtoLp0g0QsjTUbADlQ"
+}
+```
+## Set up Autolab as a tool on the LTI platform
+
+In order for Autolab to interface with LTI platforms, it will need to have a `client_id` generated for it by the platform.
+ Canvas sometimes calls their `client_id`s as "developer keys".
+### Generating a `client_id` on Canvas
+To generate a developer key, follow the guide [provided by Canvas](https://community.canvaslms.com/t5/Admin-Guide/How-do-I-configure-an-LTI-key-for-an-account/ta-p/140)
+and the instructions below. NOTE: You must be a Canvas Administrator in order to generate a developer key.
+
+During the process, fill in the JSON below with your Autolab domain, and also include your public JWK generated in the previous
+step. Then, in the step "Select Configuration Method" use the "Paste JSON" method and paste in this JSON object.
+
+Save your configuration, and then save the `client_id` generated by Canvas. This will be used to fill in the configuration file
+on Autolab's end as well.
+
+```json
+    {
+      "title": "Autolab Integration",
+      "description": "Autolab is an open-source autograding service developed by students, for students",
+      "oidc_initiation_url":"https://<your-autolab-domain>/lti_launch/oidc_login/",
+      "target_link_uri":"https://<your-autolab-domain>/lti_launch/launch/",
+      "scopes": [
+        "https://purl.imsglobal.org/spec/lti-nrps/scope/contextmembership.readonly"
+      ],
+     "extensions":[
+        {
+          "tool_id": "autolab-integration",
+          "platform": "canvas.instructure.com",
+          "domain": "<your-autolab-domain>",
+          "privacy_level": "public",
+          "settings": {
+          "text":"Launch Autolab Integration",
+          "icon_url":"<your-icon>",
+            "placements": [
+              {
+                "text":"Autolab",
+                "enabled": true,
+                "placement":"course_navigation",
+                "message_type": "LtiResourceLinkRequest",
+                "target_link_uri": "https://<your-autolab-domain>/lti_launch/launch/",
+                "icon_url":"<your-icon>",
+                "windowTarget": "_blank"
+               }
+            ]
+          }
+        }
+      ],
+      // INSERT YOUR PUBLIC JWK HERE
```

Comment:
Yeah, I was worried about users missing the part about filling in their own JWK but I guess we should assume they have enough understanding to be able to fill it in rather than break the JSON file